internal/exec/util/passwd.go

```diff
@@ -130,11 +132,20 @@ func translateV2_1UsercreateGroupSliceToPasswdUserGroupSlice(groups []types.User
 	return newGroups
 }
 
+// CheckIfUserExists will return Info log when user is empty
 func (u Util) CheckIfUserExists(c types.PasswdUser) (bool, error) {
-	code, err := u.LogCmd(exec.Command(distro.ChrootCmd(), u.DestDir, distro.IdCmd(), c.Name),
-		"checking if user %q exists", c.Name)
+	code := -1
+	cmd := exec.Command(distro.ChrootCmd(), u.DestDir, distro.IdCmd(), c.Name)
+	stdout, _ := cmd.CombinedOutput()
+	err := exec.Command(distro.ChrootCmd(), u.DestDir, distro.IdCmd(), c.Name).Run()
```

Comment:
You don't need to execute this twice, change the `_` on the previous line to be `err` and drop this line completely.

Reply:
To be more specifc: https://gist.github.com/dgonyeo/b1d9dd56edfee571b11eb2d45954130b